src/Components/Assets/AssetImportModal.tsx

```diff
@@ -43,18 +42,14 @@ const AssetImportModal = ({ open, onClose, facility }: Props) => {
     setSelectedFile(undefined);
     onClose && onClose();
   };
-
-  useEffect(() => {
-    setLocationsLoading(true);
```

Comment:
Why was this removed? The loader for when the locations are still being fetched does not work anymore.. `setLocationsLoading`